sdk/core/core-auth/src/tracing.ts

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
```

Comment:
can we put a comment in this file that it should be kept in sync with the version in core-tracing?

Reply:
Note added! Now hopefully we'll see that note when we update core-tracing...